src/router/index.js

```diff
@@ -1,3 +1,7 @@
-const routesList = []
+import { Router } from 'express'
+
+const routes = Router()
+
+const routesList = [routes.get('/', routes)]
```

Comment:
In this line you forgot to pass some function on the second param. "Routes" is wrong. Change the second param to some function, just to show that the API is running. Remove this line and add:
```
const routesList = [
  routes.get('/', function (request, response) {
    try {
      const data = { title: 'Padma Yoga API', version: '0.0.0' }
      return response.status(200).send({ data })
    } catch (error) {
      return response
        .status(500)
        .send({ message: 'Falha ao processar sua requisição', error })
    }
  }),
]
```